build_status.html

```diff
@@ -0,0 +1,353 @@
+               data-toggle="dropdown-submenu">
+              <span class="mbri-github mbr-iconfont mbr-iconfont-btn"></span>
+              GitHub
+            </a>
+            <div class="dropdown-menu">
+              <a class="text-white dropdown-item display-4"
+                 href="https://github.com/SciTools/iris"
+                 aria-expanded="true">
+                Iris
+              </a>
+              <a class="text-white dropdown-item display-4"
+                 href="https://github.com/SciTools/cartopy"
+                 aria-expanded="true">
+                Cartopy
+              </a>
+              <a class="text-white dropdown-item display-4"
+                 href="https://github.com/SciTools/cf_units"
+                 aria-expanded="true">
+                cf_units
+              </a>
+              <a class="text-white dropdown-item display-4"
+                 href="https://github.com/SciTools/iris-grib"
+                 aria-expanded="true">
+                iris-grib
+              </a>
+              <div class="dropdown-divider"></div>
+              <a class="text-white dropdown-item display-4"
+                 href="https://github.com/SciTools"
+                 aria-expanded="true">
+                All Repositories
+              </a>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </section>
+  <!-- END HEADER -->
+
+  <section class="mbr-section content4 status-title">
+    <div class="container">
+      <div class="media-container-row">
+        <div class="title col-12 col-md-8">
+          <h2 class="align-center pb-3 mbr-fonts-style display-2">
+            SciTools Build Status
+          </h2>
+          <h3 class= "mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
+            One-stop-spot for a quick summary of SciTools' health.           
+          </h3>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-table iris-cartopy-box">
+    <div class="container container-table">
+      <h3 class="mbr-section-subtitle mbr-fonts-style align-lef pb-5 mbr-light display-5">
+        The table below shows the build status of all the major packages contained in SciTools and SciTools-incubator. If available it also shows the test coverage of the package.
+      </h3>
+      <div class="table-wrapper">
+        <div class="container scroll">
+          <table class="table" cellspacing="0" id="status-table">
+            <thead>
+              <tr class="table-heads">
+                <th class="head-item mbr-fonts-style display-7">
+                  Package Name
+                </th>
+                <th class="head-item mbr-fonts-style display-7">
+                  Build Status
+                </th>
+                <th class="head-item mbr-fonts-style display-7">
+                  Test Coverage
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+
```

Comment:
Perhaps put a html comment in here to highlight that this is populated via JS at the end of the file